build-tools/packages/build-cli/src/commands/update/fluid-imports.ts

```diff
@@ -0,0 +1,332 @@
+		{ scope: "alpha", kind: "interface", name: "ISummarizerInternalsProvider" },
+		{ scope: "alpha", kind: "interface", name: "ISummarizerRuntime" },
+		{ scope: "internal", kind: "interface", name: "ISummarizingWarning" },
+		{ scope: "alpha", kind: "interface", name: "ISummary" },
+		{ scope: "alpha", kind: "interface", name: "ISummaryAckMessage" },
+		{ scope: "alpha", kind: "interface", name: "ISummaryBaseConfiguration" },
+		{ scope: "alpha", kind: "type", name: "ISummaryCancellationToken" },
+		{ scope: "alpha", kind: "interface", name: "ISummaryCollectionOpEvents" },
+		{ scope: "alpha", kind: "type", name: "ISummaryConfiguration" },
+		{ scope: "alpha", kind: "interface", name: "ISummaryConfigurationDisableHeuristics" },
+		{ scope: "alpha", kind: "interface", name: "ISummaryConfigurationDisableSummarizer" },
+		{ scope: "alpha", kind: "interface", name: "ISummaryConfigurationHeuristics" },
+		{ scope: "alpha", kind: "type", name: "ISummaryMetadataMessage" },
+		{ scope: "alpha", kind: "interface", name: "ISummaryNackMessage" },
+		{ scope: "alpha", kind: "interface", name: "ISummaryOpMessage" },
+		{ scope: "alpha", kind: "interface", name: "ISummaryRuntimeOptions" },
+		{ scope: "alpha", kind: "interface", name: "ISweepPhaseStats" },
+		{ scope: "alpha", kind: "interface", name: "IUploadSummaryResult" },
+		{ scope: "internal", kind: "const", name: "neverCancelledSummaryToken" },
+		{ scope: "alpha", kind: "type", name: "OpActionEventListener" },
+		{ scope: "alpha", kind: "type", name: "OpActionEventName" },
+		{
+			scope: "internal",
+			kind: "interface",
+			name: "RecentlyAddedContainerRuntimeMessageDetails",
+		},
+		{ scope: "internal", kind: "enum", name: "RuntimeHeaders" },
+		{ scope: "internal", kind: "enum", name: "RuntimeMessage" },
+		{ scope: "alpha", kind: "interface", name: "SubmitSummaryFailureData" },
+		{ scope: "alpha", kind: "type", name: "SubmitSummaryResult" },
+		{ scope: "alpha", kind: "class", name: "Summarizer" },
+		{ scope: "alpha", kind: "type", name: "SummarizeResultPart" },
+		{ scope: "alpha", kind: "type", name: "SummarizerStopReason" },
+		{ scope: "alpha", kind: "class", name: "SummaryCollection" },
+		{ scope: "alpha", kind: "type", name: "SummaryStage" },
+		{ scope: "alpha", kind: "const", name: "TombstoneResponseHeaderKey" },
+		{ scope: "internal", kind: "interface", name: "UnknownContainerRuntimeMessage" },
+		{ scope: "internal", kind: "function", name: "unpackRuntimeMessage" },
+	],
+	"@fluidframework/container-runtime-definitions": [
+		{ scope: "alpha", kind: "interface", name: "IContainerRuntime" },
+		{ scope: "alpha", kind: "type", name: "IContainerRuntimeBaseWithCombinedEvents" },
+		{ scope: "alpha", kind: "interface", name: "IContainerRuntimeEvents" },
+		{
+			scope: "alpha",
+			kind: "interface",
+			name: "IContainerRuntimeWithResolveHandle_Deprecated",
+		},
+	],
+};
+
+interface MemberData {
+	internal?: string[];
+	public?: string[];
+	alpha?: string[];
+	beta?: string[];
+}
+
```

Comment:
Just a note: You could also say
```typescript
type MemberData = Partial<Record<ApiLevel, string[]>>;
```
which ensures ApiLevel and MemberData agree. You'd still be able to . access those known level names.

Reply:
One of these days I'll get used to "Record". I hate the name and find it confusing, so I tend to avoid it.

Reply:
`Record` use here is really on the margins. `interface` is good if there might be things other than level lookups.